### ISSUE TITLE
将edgenext转换成ncnn报错，虽然ncnn模型转出来了但是不能正常推理，有什么解决办法吗

### ISSUE THREAD

**Issue by GloryKnight-K** [pnnx/pnnx#95]
./pnnx edgenext3.pt inputshape=[1,3,112,112]
pnnxparam = edgenext3.pnnx.param
pnnxbin = edgenext3.pnnx.bin
pnnxpy = edgenext3_pnnx.py
pnnxonnx = edgenext3.pnnx.onnx
ncnnparam = edgenext3.ncnn.param
ncnnbin = edgenext3.ncnn.bin
ncnnpy = edgenext3_ncnn.py
fp16 = 1
optlevel = 2
device = cpu
inputshape = [1,3,112,112]f32
inputshape2 = 
customop = 
moduleop = 
############# pass_level0
inline module = models.conv_encoder.ConvEncoder
inline module = models.layers.LayerNorm
inline module = models.layers.PositionalEncodingFourier
inline module = models.sdta_encoder.SDTAEncoder
inline module = models.sdta_encoder.XCA
inline module = timm.models.layers.drop.DropPath
inline module = torch.nn.modules.linear.Identity
inline module = models.conv_encoder.ConvEncoder
inline module = models.layers.LayerNorm
inline module = models.layers.PositionalEncodingFourier
inline module = models.sdta_encoder.SDTAEncoder
inline module = models.sdta_encoder.XCA
inline module = timm.models.layers.drop.DropPath
inline module = torch.nn.modules.linear.Identity

----------------

############# pass_ncnn
force batch axis 233 for operand 2
force batch axis 233 for operand 15
force batch axis 233 for operand 25
force batch axis 233 for operand 39
force batch axis 233 for operand 49
force batch axis 233 for operand 59
force batch axis 233 for operand 60
force batch axis 233 for operand 106
force batch axis 233 for operand 116
force batch axis 233 for operand 126
force batch axis 233 for operand 136
force batch axis 233 for operand 146
force batch axis 233 for operand 156
force batch axis 233 for operand 166
force batch axis 233 for operand 176
force batch axis 233 for operand 186
force batch axis 233 for operand 187
force batch axis 233 for operand 235
force batch axis 233 for operand 245
force batch axis 233 for operand 255
force batch axis 233 for operand 256
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
binaryop broadcast across batch axis 233 and 0 is not supported
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
insert_reshape_linear 4
unsupported normalize for 3-rank tensor with axis 2
unsupported normalize for 3-rank tensor with axis 2
unsupported normalize for 3-rank tensor with axis 2
unsupported normalize for 3-rank tensor with axis 2
unsupported normalize for 3-rank tensor with axis 2
unsupported normalize for 3-rank tensor with axis 2
[Uploading edgenext3.zip…]()
